src/main/resources/application.yml

```diff
@@ -113,6 +113,7 @@ queueconfig:
   uac-queue-DLQ: case.rh.uac.dlq
   uac-routing-key: event.uac.update
   response-authentication-routing-key: event.response.authentication
+  qid-filter-prefixes: 11,12,13,14
```

Comment:
Use springs list notation

qid-filter-prefixes:
  - 11
  - 12
  - 13
  - 14

Is then simpler and consistent with how CCSvc does similar

Reply:
That's lost in GH markdown rendering! see CCSvc application.yml whitelisted event property